src/components/tabs/index.tsx

```diff
@@ -66,11 +66,12 @@ export class TabContainer extends Component<TabContainerProps, TabContainerState
 
 interface TabProps {
     title: string
+    className?: string
 }
 
 export class Tab extends Component<TabProps> {
     public render(): ReactNode {
-        return <div className={styles.tab}>
+        return <div className={classNames(styles.tab, this.props.className)}>
```

Comment:
I've seen classNames() used a few times in other files and here. Is this just the way you're generating the unique ID for each element? Or is it more nuanced?

Reply:
`classNames()` is a helper I made for generating lists of class names. 

You can use it in a couple of different ways, if you pass it strings it will space separate them and ignore falsy values.

```tsx
classNames('class1', null, 'class2') === "class1 class2"
```

You can pass it an object and if the values are true it will add the corresponding keys to the class list.

```tsx
classNames({ class1: true, class2: false, class3: true }) === "class1 class3"
```

And you can combine the first 2 types into 1 call


```tsx
classNames("class1", null, { class2: true, class3: false }) === "class1 class2"
```